common/client-core/config-types/src/old/v2.rs

```diff
@@ -124,9 +137,9 @@ pub struct TrafficV1_1_20 {
     pub secondary_packet_size: Option<PacketSize>,
 }
 
-impl From<TrafficV1_1_20> for TrafficV1_1_20_2 {
-    fn from(value: TrafficV1_1_20) -> Self {
-        TrafficV1_1_20_2 {
+impl From<TrafficV2> for TrafficV3 {
```

Comment:
This feels cleaner than before, the naming that is. Nice :)

Reply:
Yep. Especially since we stopped properly versioning the client-core crate